src/core/fonts_utils.js

```diff
@@ -135,7 +138,17 @@ function type1FontGlyphMapping(properties, builtInEncoding, glyphNames) {
   let glyphId, charCode, baseEncoding;
   const isSymbolicFont = !!(properties.flags & FontFlags.Symbolic);
 
-  if (properties.baseEncodingName) {
+  if (properties.isStandardFont) {
```

Comment:
Wait, shouldn't this actually be `properties.isInternalFont` now given the change to this parameter?
This might be related to the movement in the reference tests!?